.travis.yml

```diff
@@ -1,39 +1,31 @@
-notifications: # set notification options
-  email:
-    recipients:
-      - mgopez95@gmail.com
-      - peter.kruczkiewicz@gmail.com
-
-    # change is when the repo status goes from pass to fail or vice versa
-    on_success: change
-    on_failure: change
-
 language: python
 python:
-  - '3.6'
+- '3.6'
 os:
-  - linux
-#  - osx
-
-# Whitelist of branches to run CI Testing on.
+- linux
 branches:
   only:
-    - master
-    - development
-
-# Blacklist of branches to not run CI testing on.
-# branches:
-#  except:
-#   - dontrunme 
-#   - test 
-
-before_install:
-  - sudo apt-get -qq update;
-
-install:
-  - python setup.py install
-  #Install pytest for testing.
-  - pip install pytest
-
-script:
-  - pytest
+  - master
+  - development
+install: pip install -U tox-travis
+script: tox
+deploy:
+  provider: pypi
+  distributions: sdist bdist_wheel
+  user: peterk87
```

Comment:
I'm hoping this will work for now (repo name is different than the package name), but it might be a good idea to create a phac-nml user on PyPI so there can be an official biohansel PyPI package.

Using the Travis CI Ruby command line tool, `travis login` and change the secure password
```
cat pypi-password | travis encrypt --add deploy.password
```